### ISSUE TITLE
Vary solar panel power production based on environment

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/OPHD#494]
Is there any desire to vary the power produced from solar panels and solar satellites?

I was thinking it would be easy to assign each planet a mean distance from star in AU (astronomical units). This could be used to scale power based on planet/moon selected, which would translate to focusing on fusion or solar depending on the effectiveness.

A lot more could be added to the calculation like planetary atmospheric conditions and energy emitted by star, but I was thinking for now just keeping it simple and scaling based on distance.

To implement, I would add a constants section for starting planet with this value and maybe move hostility into it. Longer term, it might be nice to have a larger set of parameters to change gameplay based on planetary settings.

-Brett

**Comment by DanRStevens** [comment]
Yes, I believe there was a plan to vary solar power, and to have that setting impacted by planet selection and difficulty. I don't believe any specific details were decided yet though.

**Comment by ldicker83** [comment]
Dan linked to the general area. A bit earlier I explain the overall idea I have for setting difficulty and simulation parameters. https://github.com/OutpostUniverse/OPHD/issues/200#issuecomment-606706035

But basically yes, the idea is that at some point environment setup will have an effect on solar panel generation (and solar satellite generation when that technology is developed).

**Comment by Brett208** [comment]
OK, sounds good, I'll try to add mean solar distance to the game and use it to make Mars get the most power and Gandymede the least for solar cells and then close this issue. Other updates can get more complex in adding other variables such as chosen star, etc.

This issue is Semi-Blocked by issue #506

**Comment by ldicker83** [comment]
I would imagine the Mercury type would get the most solar illumniation.

**Comment by Brett208** [comment]
This is fully implemented now.
